Modules/System/SharePoint/src/SharePointClientImpl.Codeunit.al

```diff
@@ -589,6 +596,68 @@ codeunit 9101 "SharePoint Client Impl."
         SharePointFileParser.ParseSingleReturnValue(Result, SharePointFile);
         exit(true);
     end;
+
+    procedure UpdateListItemMetaDataField(ListTitle: Text; ItemId: Integer; ListItemEntityTypeFullName: Text; FieldName: Text; FieldValue: Text): Boolean
```

Comment:
If I understand this correctly then this procedures is using the API v1.
https://learn.microsoft.com/en-us/sharepoint/dev/sp-add-ins/working-with-folders-and-files-with-rest
Does anybody know if  all methods are using the API v1 or if there are methods, that are using the API v2 (graph API)?
Maybe we can document which procedures we are using by referencing online documentation of the API, if documention is available.